monitoring/monitorlib/infrastructure.py

```diff
@@ -201,23 +205,23 @@ def wrapper_default_scope(*args, **kwargs):
       # Change <DSSTestSession>.default_scopes to scopes for all DSSTestSession arguments
```

Comment:
Looks like there are few remaining occurrence of DSSTestSession in comments. (9 occurences in [monitorlib/infrastrucure.py](https://github.com/interuss/dss/blob/566fc0b3f50f2460709cc3db1cf73bc20ddbd17e/monitoring/monitorlib/infrastructure.py) and one in [introduction_to_repository.md](https://github.com/interuss/dss/blob/master/introduction_to_repository.md?plain=1#L37))

Reply:
Good catch -- thanks